device-types/Datacom/DM4770-16CX.yaml

```diff
@@ -0,0 +1,91 @@
+---
+manufacturer: Datacom
+model: DM4770 16CX
+slug: datacom-dm4770-16cx
+part_number: 800.5280.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: "Switch with L2, L3, and MPLS capabilities, with 12 40GE/100GE interfaces\non QSFP28 connectors, 4 40GE/100GE interfaces on QSFP-DD connectors,\
+  \ and 4\n10GE/25GE interfaces on SFP28 connectors. 1U metal enclosure for 19 rack with one slot for ventilation module (included) and two slots for redundant\
+  \ AC or DC power supplies. Outband management interfaces, RS232 console, and USB ports.\nPower supplies, MPLS license, and optical modules must be purchased\
+  \ separately."
+weight: 4
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+console-server-ports:
+  - name: Storage
+    type: usb-a
```

Comment:
Is this actually used as a console server port? Or is it just for the switch to access a USB storage device? If the latter, please remove this definition as it would be out of scope.

Reply:
removed